src/postmate.js

```diff
@@ -70,7 +70,7 @@ export const sanitize = (message, allowedOrigin) => {
     typeof allowedOrigin === 'string' &&
     message.origin !== allowedOrigin
   ) return false
-  if (typeof message.data !== 'object') return false
+  if (!message.data) return false
```

Comment:
Would you not prefer to check strictly against `null` equality, @Andarist?

Reply:
i thought about it, decided that truthy check is probably good enough here, if u want i might change it to
```js
if (typeof message.data !== 'object' || message.data === null) return false
```